storage/src/mmr/bitmap.rs

```diff
@@ -723,10 +732,9 @@ mod tests {
         let executor = deterministic::Runner::default();
         executor.start(|context| async move {
             // Initializing from an empty partition should result in an empty bitmap.
-            let mut bitmap =
-                Bitmap::<Sha256, 32>::restore_pruned(context.clone(), PARTITION.to_string())
-                    .await
-                    .unwrap();
+            let mut bitmap = Bitmap::<Sha256, 32>::restore_pruned(context.clone(), PARTITION)
```

Comment:
Is there some constant we should be using here? I think Sha256::Digest::SIZE should exist (for example)?